### ISSUE TITLE
Version 8.0.0 causes Type E_ERROR when in use with PayPal Payments.

### ISSUE THREAD

**Issue by ryanr14** [Automattic/woocommerce-payments#9195]
### Describe the bug
If you are using PayPal Payments and WooPayments, updating WooPayments to 8.0.0 will cause the following error. It does seem like rolling back to 7.9.2 fixes things for the time being.

```
An error of type E_ERROR was caused in line 47 of the file /srv/htdocs/wp-content/plugins/woocommerce-paypal-payments/modules/ppcp-blocks/extensions.php. Error message: Uncaught Error: Call to undefined function WooCommerce\PayPalCommerce\Blocks\wc_terms_and_conditions_page_id() in /srv/htdocs/wp-content/plugins/woocommerce-paypal-payments/modules/ppcp-blocks/extensions.php:47
```

### To Reproduce

1. Activate PayPal Payments
2. Update WooPayments to 8.0.0
3. See error.

### Additional context

https://wordpress.org/support/topic/site-crashes-after-update-4/

**Comment by Bentipa** [comment]
+1 can confirm, rolling back to 7.9.2 solves the issue

**Comment by aisajib** [comment]
Just moving it over from the Slack thread: p1722429988062499-slack-C7U3Y3VMY - These are the two reported cases I had a while ago.

8554798-zd-a8c
8551209-zd-a8c

**Comment by marcuswisecaesar** [comment]
8554907-zd-a8c

**Comment by kdingsa8c** [comment]
8554760-zd-a8c
8555463-zd-a8c

**Comment by fabienb** [comment]
Not just any error, it's a fatal error. 
This time it seems to be broader than the crippling performance issue with Litespeed servers caused only last month... Perhaps implement better testing before deployment?

**Comment by tvolpert** [comment]
8555187-zd-a8c
8555424-zd-a8c
8555487-zd-a8c

**Comment by ccwalburn** [comment]
8555398-zd-a8c

**Comment by ClassicRKR27** [comment]
I have another instance here: 8555629-zd-a8c

This one is a bit strange though as trying to revert the plugin version for `WooPayments` didn't work in this case, I had to completely disable `WooCommerce PayPal Payments` in order to get it to work at all - not sure if there is something different about this particular setup but it's probably worth investigating further.

Edit: I actually found that the CLI command doesn't appear to be working at all anymore for some reason, more details here: p1722441491423049/1722437876.296799-slack-CGGCLBN58

**Comment by ezebecke** [comment]
8555680-zd-a8c

**Comment by ClassicRKR27** [comment]
Another instance: 8555682-zd-a8c

**Comment by ariel-maidana** [comment]
Another instance: 8555665-zd-a8c

**Comment by Joshgrygiel** [comment]
Another: 8555704-zd-a8c

**Comment by Joshgrygiel** [comment]
Another: 8555707-zd-a8c

**Comment by susanjanec** [comment]
Another instance here - 8555714-zd-a8c

**Comment by ariel-maidana** [comment]
Another instance: 8555712-zd-a8c

**Comment by Joshgrygiel** [comment]
Another: 8555744-zd-a8c

**Comment by tavilla61** [comment]
Another one:  8555734-zd-a8c

**Comment by susanjanec** [comment]
Another - 8555845-zd-a8c

**Comment by ezebecke** [comment]
8555835-zd-a8c
8555884-zd-a8c

**Comment by c-shultz** [comment]
This team [has prepared a fix](https://github.com/Automattic/woocommerce-payments/issues/9196) and is actively testing/deploying it now.

**Comment by TeniCola** [comment]
Several more cases:

8555856-zd-a8c
8555832-zd-a8c
8555806-zd-a8c
8555818-zd-a8c

**Comment by khristiansnyder** [comment]
8555901-zd-a8c

**Comment by thomasjvu** [comment]
8555719-zd-a8c

**Comment by kdingsa8c** [comment]
8555878-zd-a8c

**Comment by tanjoymor** [comment]
8538840-zd-a8c

**Comment by paymentplugins** [comment]
Looks like you will be releasing an update soon, but here is a support thread which adds some additional info and the reason for the issue.

[https://wordpress.org/support/topic/woopayments-8-0-kills-3rd-party-payment-methods/#post-17925382](https://wordpress.org/support/topic/woopayments-8-0-kills-3rd-party-payment-methods/#post-17925382)

**Comment by cardehngeh** [comment]
8555458-zen
8555802-zen

**Comment by oaratovskyi** [comment]
Closed in #9196 (same bug, two issues)

**Comment by oaratovskyi** [comment]
oh, and it was released in 8.0.1
